northcoders/frontend/games-project/src/components/Nav.jsx

```diff
@@ -0,0 +1,29 @@
+import { Link } from "react-router-dom";
+import { useEffect, useState } from "react";
+import{ getCategories } from "../utils/api";
+
+const Nav = () =>{
+    const [categories, setCategories] = useState([]);
+    
+    useEffect(()=>{
+        getCategories()
+        .then(({categories})=>{
+            setCategories(categories)
+        })
+    },[])
+
+
+return <ul>
```

Comment:
Realy like the UL here however you arn't using any list items, you may want to wrap your nav links inside LI's